src/VisualStudio/Core/Def/ServicesVSResources.resx

```diff
@@ -1813,6 +1813,9 @@ Additional information: {1}</value>
   <data name="Run_code_analysis_in_separate_process_requires_restart" xml:space="preserve">
     <value>Run code analysis in separate process (requires restart)</value>
   </data>
+  <data name="Run_code_analysis_on_dotnet_requires_restart" xml:space="preserve">
+    <value>Run code analysis on .NET (requires restart)</value>
```

Comment:
I can get the desire to not say ".NET Core", but this text might be a bit of a head scratcher for folks if they think unchecking this means we're running it on, uh, C++? Should we just say ".NET 6" or something so it's a bit clearer that way?

Reply:
How about " latest .NET"? Given that we might be moving to .NET 7 soon

Reply:
<img width="352" alt="image" src="https://github.com/dotnet/roslyn/assets/788783/cf1a4ddd-a6be-482c-973a-2aefdc16d15c">

Reply:
@genlu Oh I like that!